rest_framework_nested/runtests/runcoverage.py

```diff
@@ -34,7 +36,7 @@ def main():
             'Function-based test runners are deprecated. Test runners should be classes with a run_tests() method.',
             DeprecationWarning
         )
-        failures = TestRunner(['tests'])
+        failures = TestRunner(['tests'])  # type: ignore[assignment,arg-type]
```

Comment:
I don't think this code path even works. `TestRunner` doesn't seem to accept list argument.

Reply:
Me neither.